docs/source/example_notebooks/dowhy_causal_discovery_example.ipynb

```diff
@@ -0,0 +1,692 @@
+      "No such variable found!\n",
+      "\n",
+      "### Estimand : 3\n",
+      "Estimand name: frontdoor\n",
+      "Estimand expression:\n",
+      "Expectation(Derivative(PKC, [plcg])*Derivative([plcg], [PIP2]))\n",
+      "Estimand assumption 1, Full-mediation: plcg intercepts (blocks) all directed paths from PIP2 to P,K,C.\n",
+      "Estimand assumption 2, First-stage-unconfoundedness: If U→{PIP2} and U→{plcg} then P(plcg|PIP2,U) = P(plcg|PIP2)\n",
+      "Estimand assumption 3, Second-stage-unconfoundedness: If U→{plcg} and U→PKC then P(PKC|plcg, PIP2, U) = P(PKC|plcg, PIP2)\n",
+      "\n",
+      "Causal Estimate is 0.10456597412424884\n",
+      "\n",
+      "*****************************************************************************\n",
+      "\n",
+      "Causal Discovery Method : GES\n",
+      "Estimand type: nonparametric-ate\n",
+      "\n",
+      "### Estimand : 1\n",
+      "Estimand name: backdoor\n",
+      "Estimand expression:\n",
+      "   d                             \n",
+      "───────(Expectation(PKC|PKA,P38))\n",
+      "d[PIP₂]                          \n",
+      "Estimand assumption 1, Unconfoundedness: If U→{PIP2} and U→PKC then P(PKC|PIP2,PKA,P38,U) = P(PKC|PIP2,PKA,P38)\n",
+      "\n",
+      "### Estimand : 2\n",
+      "Estimand name: iv\n",
+      "No such variable found!\n",
+      "\n",
+      "### Estimand : 3\n",
+      "Estimand name: frontdoor\n",
+      "No such variable found!\n",
+      "\n",
+      "Causal Estimate is -0.013018300750143652\n"
+     ]
+    }
+   ],
+   "source": [
+    "for method, graph in graphs.items():\n",
+    "        print('\\n*****************************************************************************\\n')\n",
+    "        print(\"Causal Discovery Method : %s\"%(method))\n",
+    "\n",
+    "        # Obtain valid dot format\n",
+    "        graph_dot = str_to_dot(graph.source)\n",
+    "\n",
+    "        # Define Causal Model\n",
+    "        model=CausalModel(\n",
+    "                data = data_sachs,\n",
+    "                treatment='PIP2',\n",
+    "                outcome='PKC',\n",
+    "                graph=graph_dot)\n",
+    "\n",
+    "        # Identification\n",
+    "        identified_estimand = model.identify_effect(proceed_when_unidentifiable=True)\n",
+    "        print(identified_estimand)\n",
+    "\n",
+    "        # Estimation\n",
+    "        estimate = model.estimate_effect(identified_estimand,\n",
+    "                                        method_name=\"backdoor.linear_regression\",\n",
+    "                                        control_value=0,\n",
+    "                                        treatment_value=1,\n",
+    "                                        confidence_intervals=True,\n",
+    "                                        test_significance=True)\n",
+    "        print(\"Causal Estimate is \" + str(estimate.value))"
+   ]
+  },
+  {
+   "source": [
+    "## Graph Refutation\n",
```

Comment:
This is a great section. Need to split into two:
1. Graph refutation: just comparing the available graphs, without knowledge of true graph.
2. Graph validation: now comparing them to the ground-truth (unknown to the method) causal graph. 
Again more interpretation of the refutation will be helpful.

Reply:
Sure. Will do that.